### DISCUSSION TITLE
Using project fields loses us the ability to filter issues

### DISCUSSION THREAD

**Discussion by rptb1** [community/community#50067] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

We're using issue keywords to prioritize issues.  That allows us to filter by priority in the Issues view of our repository.

Your Projects would allow us to add a Single Select field for priority.  That makes a lot of sense.  What's more, we can have priority per project.  Great.

But if we switch over to using a priority field in a Project, we can no longer:

1. see priority in the issues view (unlike keywords)
2. filter by priority in the issues view (unlike keywords)
3. assign priority in the issues view (unlike keywords)
4. etc.

Basically, I'd like the issues view and Projects spreadsheet to be two ways of looking at the same data, not duplicate systems.

I wonder if you have separate internal teams working on these things and might be victims of Conway's Law:

> Any organization that designs a system (defined broadly) will produce a design whose structure is a copy of the organization's communication structure.

Thanks!
